src/requirements/data/colleges/asFA2020.ts

```diff
@@ -1,24 +1,42 @@
 import { Course, CollegeOrMajorRequirement } from '../../types';
-import { courseIsFWS, ifCodeMatch, courseIsForeignLang } from '../checkers-common';
+import {
+  courseIsFWS,
+  ifCodeMatch,
+  courseIsForeignLang,
+  courseCountsForASCredit,
+} from '../checkers-common';
 
 const casFA2020Requirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: '120 Total Credits',
```

Comment:
We already have a 120 total credit check in the pre-FA20 A&S college. Not sure why it never got implemented here, but could you model this one after that one to be consistent?

Reply:
deleted this and copied the pre2020 one

Reply:
it's not as accurate because it circumvents our usual way of checking requirements, but it'll do for now